ReSharper.FSharp/test/data/features/service/codeFormatter/Match lambda expr alignment 01 - Correct alignment.fs.gold

```diff
@@ -0,0 +1,2 @@
+﻿let f = function
+        | _ -> ()
```

Comment:
Should add indent relatively the outer indentation:
```
let f = function
  | _ -> ()
```

It's good to check what indentation is allowed by looking at LexFilter module in the compiler.